src/main/java/com/dreammaster/gthandler/recipes/ElectrolyzerRecipes.java

```diff
@@ -8,293 +8,491 @@
+                GT_Values.NI,
+                GT_Values.NF,
+                Materials.Oxygen.getGas(1000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Lithium, 2L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 5L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 12L),
+                GT_Values.NI,
+                GT_Values.NI,
+                GT_Values.NI,
+                null,
+                180,
+                90);
+        // BlueTopaz
         GT_Values.RA.addElectrolyzerRecipe(
                 GT_OreDictUnificator.get(OrePrefixes.dust, Materials.BlueTopaz, 13L),
-                ItemList.Cell_Empty.get(8L),
+                ItemList.Cell_Empty.get(3L),
                 GT_Values.NF,
                 Materials.Fluorine.getGas(2000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 2L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 1L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 5L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 3L),
                 GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Hydrogen, 2L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 6L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 1L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
                 208,
                 120);
+        // Topaz
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Pollucite, 7L),
-                ItemList.Cell_Empty.get(2L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Topaz, 13L),
+                ItemList.Cell_Empty.get(3L),
                 GT_Values.NF,
-                Materials.Water.getFluid(1000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Caesium, 1L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 1L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 2L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 2L),
+                Materials.Fluorine.getGas(2000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 5L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 3L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Hydrogen, 2L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 1L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
-                210,
+                160,
                 120);
+        // Clay
```

Comment:
this is not the clay chem formula. and why 25?

Reply:
fixed